test/paymaster/singleton/verifying-singleton-paymaster.ts

```diff
@@ -69,24 +69,24 @@ describe("EntryPoint with VerifyingPaymaster", function () {
         offchainSignerAddress
       );
 
-    smartWalletImp = await new SmartAccount__factory(deployer).deploy();
-
-    maliciousWallet = await new MaliciousAccount__factory(deployer).deploy();
-
-    walletFactory = await new SmartAccountFactory__factory(deployer).deploy(
-      smartWalletImp.address
-    );
-
     callBackHandler = await new DefaultCallbackHandler__factory(
       deployer
     ).deploy();
 
-    await walletFactory.deployCounterFactualWallet(
-      walletOwnerAddress,
-      entryPoint.address,
-      callBackHandler.address,
-      0
+    smartWalletImp = await new SmartAccount__factory(deployer).deploy(
```

Comment:
is it just another way to do the same as

```
const BaseImplementation = await ethers.getContractFactory("SmartAccount");
smartWalletImp = await BaseImplementation.deploy(entryPoint.address);
await smartWalletImp.deployed()
```
?

Reply:
Yes